### PR TITLE
Adds caramel

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Adds caramel to the game. Makes the name of secret sauce consistent with other chemicals.

Adjusts the recipe of candied apples.

Side-effect is that you have to pay slightly more attention when mixing friendly life.

## Why It's Good For The Game

Caramel is delicious and the recipe is a classic.

## Changelog
:cl:
add: You can now make caramel by heating sugar. Be careful not to overcook it.
add: Adjusted the candied apple recipe
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
not really relevant but speaking of secret sauce is it broken? The recipe I got didn't work, idk whats up with that (I got the current weekly recipe)

**Comment by bgobandit** [comment]
only if you also add caramel apples

**Comment by OverlyGeneric** [comment]
Finally, we can add Snickers into the game and make it revert transformations like changelings and hulks because you're not you when you're hungry.

**Comment by FantasticFwoosh** [comment]
> 
> 
> Finally, we can add Snickers into the game and make it revert transformations like changelings and hulks because you're not you when you're hungry.

Contains trace mutadone of course, im not heavily versed in confectionary cuisine, but caramel is a good place to start for a 'sweets' category.

**Comment by ATH1909** [comment]
While I do like the idea of adding caramel to the game, I don't like how this makes activating the life reaction with the minimum doses much, much more of a pain.

**Comment by nemvar** [comment]
So ehrm... apparently we already got candied apples...

**Comment by FantasticFwoosh** [comment]
> 
> 
> I don't like how this makes activating the life reaction with the minimum doses much more of a pain (you can't just spam-click on a beaker that contains all but one of the reactants with a lighter and then transfer 1u of reagents to it at a time.

I don't understand the complaint, you can speed heat beakers with a welder or equivilent & use the chemical heater to also cool-off beakers, its so simple the chef or any ghetto chemist can do it.

**Comment by ExcessiveUseOfCobblestone** [comment]
this thread will be merged or closed depending on if you say cara-mel or car-mel

**Comment by nemvar** [comment]
Does Karamell count?

**Comment by HomocidlePancakes** [comment]
Who the fuck spells it as car-mel?

**Comment by ExcessiveUseOfCobblestone** [comment]
it's not spelling it's how you pronounce it.

**Comment by Shadowflame909** [comment]
I pronounce it car-mel

File: code/modules/food_and_drinks/recipes/food_mixtures.dm [hunk 1]
```diff
@@ -76,6 +76,22 @@
 	required_reagents = list(/datum/reagent/consumable/corn_starch = 1, /datum/reagent/toxin/acid = 1)
 	required_temp = 374
 
+/datum/chemical_reaction/caramel
+	name = "Caramel"
+	id = /datum/reagent/consumable/caramel
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/food_and_drinks/recipes/food_mixtures.dm]: is this suppose to be a string?

Comment (nemvar) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm]: There isn't any consistency on reaction ids. Some are strings, some are the type. Apparently they just have to be unique and this is just that.

Comment (ExcessiveUseOfCobblestone) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm]: lol ok

Comment (ExcessiveUseOfCobblestone) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm]: easy gbp for whoever makes them either or

**Pull request merged**